tests/modules/block_keyspace_notification.c

```diff
@@ -0,0 +1,148 @@
+
+/* This module is used to test blocking the client during a keyspace event. */
+#define _BSD_SOURCE
+#define _DEFAULT_SOURCE /* For usleep */
+#include "valkeymodule.h"
+#include <stdio.h>
+#include <string.h>
+#include <strings.h>
+#include <pthread.h>
+#include <unistd.h>
+#define EVENT_LOG_MAX_SIZE 1024
+static pthread_mutex_t event_log_mutex = PTHREAD_MUTEX_INITIALIZER;
+typedef struct KeyspaceEventData {
```

Comment:
please fix the format first, thest test codes (.c / .tcl) need some spaces between them